bootstrap.py

```diff
@@ -397,12 +397,13 @@ def clean_katello_agent():
     delete_file("/etc/rhsm/ca/katello-server-ca.pem")
 
 
-def install_katello_agent():
```

Comment:
Given `katello-agent` is still required for errata installation without REX, I wonder if we should leave that function in and add some `--install-katello-agent` swtich.

Just installing the package is not enough in the case you're migrating from one Katello install to another, as the pkg might be already installed (→ NOOP) but the daemon is not restarted for the new config.

Reply:
To migrate from one Katello install to another, you must call bootstrap.py with `--force`, which also means the `clean_katello_agent` function gets called. See https://github.com/Katello/katello-client-bootstrap/blob/master/bootstrap.py#L1328 and https://github.com/Katello/katello-client-bootstrap/blob/master/bootstrap.py#L259:L266

Reply:
I seem to remember that some older builds of `katello-agent` neither (re)started nor enabled the `goferd` daemon